### PR TITLE
Feature - add filter type for Milvus DB

### PR DESCRIPTION
**Pull request by GalO005**
A filter type has been added to Milvus DB, so we can pass filter object with a property called `filterStr`

### CODE REVIEW

File: langchain/src/vectorstores/milvus.ts [hunk 1]
```diff
@@ -223,6 +232,7 @@ export class Milvus extends VectorStore {
       output_fields: outputFields,
       vector_type: DataType.FloatVector,
       vectors: [query],
+      filter: filterStr,
```
Comment (jacoblee93) [inline langchain/src/vectorstores/milvus.ts:235]: Just checking - passing an empty string has the same behavior as undefined right?

Comment (GalO005) [reply langchain/src/vectorstores/milvus.ts:235]: Yeah, sure.

Comment (jacoblee93) [reply langchain/src/vectorstores/milvus.ts:235]: Hey actually it looks like Milvus just accepts a string: 

https://milvus.io/docs/search.md#Conduct-a-vector-search

I think we can just have `FilterType: string` here - if they add more complex objects in the future we can shim it.

Comment (GalO005) [reply langchain/src/vectorstores/milvus.ts:235]: No problem, but I think that `FilterType` is defined as object type (and thus I created interface), I can add to base class `object | string` what do you think about that?

Comment (GalO005) [reply langchain/src/vectorstores/milvus.ts:231]: @jacoblee93 I changed it.

**Comment by jacoblee93** [comment]
Thank you!

**Pull request merged**